### DISCUSSION TITLE
Clinvar module version 2024.08.01 appears to have lost ClinvarID as well as the link to Clinvar online database.

### DISCUSSION THREAD

**Discussion by clinical-ngs** [KarchinLab/open-cravat#283] [Usability & Feature Requests]
**Describe the bug**
Clinvar version 2024.08.01 appers to have lost ClinvarID as well as the underlying link to Clinvar online database. 

**To Reproduce**
1. Update to the latest clinvar module (2024.08.01)
2. Annnotate a VCF file (we use Dragen Illumina  4.2 or GATK 4.5 to generate them) with a set of modules including Clinvar.
3. Open an annotated VCF and expand Clinvar from the + sign.
4. ClinvarID field is empty. ClinvarID field has no HTML link to Clinvar website. 

**Expected behavior**
ClinvarID field should include the specific ID number.
ClinvarID should have a HTML link to Clinvar website

**Screenshots**
![Clinvar](https://github.com/user-attachments/assets/5c2c69d0-00c9-4278-a973-97e414a95dc5)

**Desktop (please complete the following information):**
 - OS: Windows 10.
 - Browser Chrome
 - Version 127.0.6533.89 (Official Build) (64-bit)

**Smartphone (please complete the following information):**

**Additional context**
ClinvarID and the link to the online Clinvar database is essential for clinical analysis. This is the one of the essential features of OC.  
The workaround is to use the rs ID and then use to link to Clinvar, if existing.

**Comment by skanderson** [comment]
Thanks for catching this! We added a lot of data to the ClinVar annotator and it looks like I forgot to include the ID field in the annotation. I just published a corrected version (2024.08.09), so you should be able to run `oc module update clinvar` to get the updated version with the ID and link fixed.
